rainier-core/src/main/scala/com/stripe/rainier/core/Predictor.scala

```diff
@@ -93,8 +93,14 @@ object Predictor {
     def fromVector[L](k: Int)(fn: IndexedSeq[U] => L): Predictor[Seq[X], L]
   }
 
-  def apply[X](implicit enc: Encoder[X]) =
-    new From[X, enc.U] {
+  trait EncoderFrom[X, U] extends From[X, U] {
```

Comment:
It MIGHT just be the case that the return types of `From`'s methods should be `EncoderPredictor` already... is that the case?

Reply:
Yes, that is in fact the case, and that would be a better change here.

Reply:
Okay, excellent, done.